.github/ISSUE_TEMPLATE/tool-request.md

```diff
@@ -32,5 +33,8 @@ assignees: ''
 **Can this tool be installed during the build?**
 <!--- If so, please provide a description of how and approximately how much time it takes.  -->

```

Comment:
Looks like we already have a question about installation time

Reply:
Agree, I have changed tip for this question. We have added separate step for approximate time, since this point has been ignored many times.